impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -35,6 +36,356 @@ define(["./_focus"], function(focusUtil) {
     delete: 46
   };
 
+  /*
+   * This software or document includes material copied from or derived from https://www.w3.org/WAI/content-assets/wai-aria-practices/patterns/listbox/examples/js/listbox.js
+   * Copyright © 2023 World Wide Web Consortium. https://www.w3.org/Consortium/Legal/2023/doc-license
+   *
+   */
+  function makeAccessibleListbox(elem) {
+    elem.setAttribute("role", "listbox");
+    elem.setAttribute("tabindex", "0");
+    elem.setAttribute("aria-activedescendant", "");
+    elem.setAttribute("aria-multiselectable", "true");
+    var activeDescendantId = null;
+    var startRangeIndex = 0;
+    var keysSoFar = "";
+    var clearKeysSoFarHandle = null;
+
+    elem.addEventListener("keydown", onKeyPress);
+    elem.addEventListener("click", onClick);
+
+    return {destroy, clearSelectedOptions, clearActiveOption, getSelectedOptions};
```

Comment:
Another case of great, new JS constructs. Even though we are not unit testing this file, we should, and don't want to block that ability by using syntactical constructs that the unit test JS environment, PhantomJS used in this project does not support.